pom.xml

```diff
@@ -39,7 +39,8 @@
     <maven.compiler.target>1.7</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-    <version.io.opentracing>0.20.7</version.io.opentracing>
+    <version.com.jayway.awaitility-awaitility>1.7.0</version.com.jayway.awaitility-awaitility>
+    <version.io.opentracing>0.20.8-SNAPSHOT</version.io.opentracing>
```

Comment:
This won't be merged with snapshot (even see CI failed). I just need latest features.